src/solvers/dg_curved/dg_3d.jl

```diff
@@ -88,26 +89,65 @@ function calc_interface_flux!(u::AbstractArray{<:Any,5}, mesh::CurvedMesh{3},
     # Interfaces in x-direction (`orientation` = 1)
     calc_interface_flux!(elements.surface_flux_values,
                          elements.left_neighbors[1, element],
-                         element, 1, u, mesh, equations, dg, cache)
+                         element, 1, u, surface_flux, mesh, equations, dg, cache)
     
     # Interfaces in x-direction (`orientation` = 2)
     calc_interface_flux!(elements.surface_flux_values,
                          elements.left_neighbors[2, element],
-                         element, 2, u, mesh, equations, dg, cache)
+                         element, 2, u, surface_flux, mesh, equations, dg, cache)
 
     # Interfaces in x-direction (`orientation` = 3)
     calc_interface_flux!(elements.surface_flux_values,
                          elements.left_neighbors[3, element],
-                         element, 3, u, mesh, equations, dg, cache)
+                         element, 3, u, surface_flux, mesh, equations, dg, cache)
   end
 
   return nothing
 end
 
 
 @inline function calc_interface_flux!(surface_flux_values, left_element, right_element, orientation, u, 
-                              mesh::CurvedMesh{3}, equations, dg::DG, cache)
-  @unpack surface_flux = dg
```

Comment:
Why can't you just keep unpacking the `surface_flux` from `dg` as before??

Reply:
We used the `surface_flux` to dispatch if it's `FluxRotated` or not. So, in case of no `FluxRotated` (e.g. LSA) we don't need to initialize the tangent vectors and can call the surface flux without these not needed parameters.
Otherwise, we would have a general flux function with the normal vector and both tangent vectors. Then we have to add a `FluxPlusDissipation` constructor with the unnecessary parameters.

Reply:
OK, then just leave it as it is. The holw `FluxRotated` business is not yet fully converged anyways, so no noeed to spend more time than necessary on this now.